src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java

```diff
@@ -191,6 +191,7 @@
     private volatile ClusterService cs;
     private volatile AuditLog auditLog;
     private volatile BackendRegistry backendRegistry;
+    private volatile CompatConfig compatConfig;
```

Comment:
Why it is necessary to make `compatConfig` member variable?

Reply:
Will revert. I initially required it because it was being used in multiple places, but not required anymore.